FiveCalls/FiveCalls/AppDelegate.swift

```diff
@@ -31,22 +26,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         oneSignalStartup(launchOptions: launchOptions)
         OneSignal.setExternalUserId(AnalyticsManager.shared.callerID)
         
-        window = UIWindow()
```

Comment:
🎉 eventually, we can probably migrate all of this out of the AppDelegate, right?

Reply:
Maybe in a few iOS versions? 😅 I don't think `App` supports 100% of what `UIApplicationDelegate` can handle yet